web/gds-admin-ui/src/components/DeleteModal.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import ButtonSpinner from './ButtonSpinner';
+import warningImg from 'assets/images/warning.svg'
+import { ModalCloseButton } from './Modal';
+import PropTypes from 'prop-types'
+
+function DeleteModal({ onDelete, isLoading = false }) {
+
+    return <div className='text-center'>
+        <img src={warningImg} alt="Warning" />
+        <h4>Are you sure?</h4>
```

Comment:
Can we change the heading message to something like the following?

`Are you sure you want to delete registration <registration_name>?`

We want to make it as clear as possible to the user what they are deleting.

Reply:
you right